### PR TITLE
Add Travis CI - Tests

### PR DESCRIPTION
**Pull request by dubee**
- Run automation via Travis

Closes https://github.com/openwhisk/openwhisk-package-kafka/issues/109

### CODE REVIEW

File: tools/travis/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+set -e
+
+# Build script for Travis-CI.
+
+SCRIPTDIR=$(cd $(dirname "$0") && pwd)
+ROOTDIR="$SCRIPTDIR/../.."
+
+cd $ROOTDIR
+docker build -t kafka_feed_provider .
+(docker run -i -t -e CLOUDANT_USER="" -e CLOUDANT_PASS="" -e LOCAL_DEV=true -e GENERIC_KAFKA=false -p 5000:5000 kafka_feed_provider) &
```
Comment (bjustin-ibm) [inline tools/travis/build.sh]: What's the plan for the Cloudant username and password?

**Comment by bjustin-ibm** [comment]
@dubeejw Lets make sure we include the scanCode script as well: https://github.com/openwhisk/openwhisk-catalog/blob/master/tools/travis/build.sh#L13

File: tests/build.gradle [hunk 2]
```diff
@@ -71,10 +71,16 @@ tasks.withType(ScalaCompile) {
   scalaCompileOptions.additionalParameters = gradle.scala.compileFlags
 }
 
+task testProvider(type: Test) {
```
Comment (jasonpet) [inline tests/build.gradle]: what is this?

File: tests/build.gradle [hunk 3]
```diff
@@ -71,10 +71,16 @@ tasks.withType(ScalaCompile) {
...
+  systemProperty 'test.router', 'true'
```
Comment (jasonpet) [inline tests/build.gradle]: why do you need this?

**Comment by bjustin-ibm** [comment]
@dubeejw Let's shift focus on this to just getting the scanCode script running ASAP. We can add tests after. Feel free to use a separate PR to just get scanCode running.

**Comment by rabbah** [comment]
Closing as out of date and abandoned.

**Pull request closed without merging**